.gitignore

```diff
@@ -19,3 +19,5 @@
 # Drupal
 /core
 
+
+.idea
```

Comment:
@alex-arriaga: We like to keep the IDE and other user files using the global .gitignore feature:
https://help.github.com/articles/ignoring-files/

`git config --global core.excludesfile ~/.gitignore_global`